scripts/pulsar/prepare_helm_release.sh

```diff
@@ -33,6 +33,7 @@ Options:
        --control-center-password        the password of control center administrator
        --pulsar-superusers              the superusers of pulsar cluster. a comma separated list of super users.
        -c,--create-namespace            flag to create k8s namespace.
+       --service-account-key-file       the path of service account key file.
```

Comment:
Isn't the service account key file only applied to GCS? Does it work for S3? If it only works for GCS, please use a more explicit name as the option name and add the document.

Besides that, since we are creating the service account key as secret for GCS, why not also provide an option to create a secret for S3?

Reply:
> Isn't the service account key file only applied to GCS? Does it work for S3?

Yes, the `service-account-key-file` only applied to GCS.

> Besides that, since we are creating the service account key as a secret for GCS, why not also provide an option to create a secret for S3?

In S3, we can config the secret by `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` in `values.yaml`